src/metabase/models/notification.clj

```diff
@@ -151,7 +151,8 @@
 (def ^:private notification-recipient-types
   #{:notification-recipient/user
     :notification-recipient/group
-    :notification-recipient/external-email})
+    :notification-recipient/external-email
+    :notification-recipient/params})
```

Comment:
What do you think about calling this `:notification-recipient/template` instead?